packager/react-native-xcode.sh

```diff
@@ -33,6 +33,14 @@ cd ..
 set -x
 DEST=$CONFIGURATION_BUILD_DIR/$UNLOCALIZED_RESOURCES_FOLDER_PATH
 
+# Define NVM_DIR and source the nvm.sh setup script
+export NVM_DIR=$HOME/.nvm
```

Comment:
Yep - I'll add an little `-z` check. (let me know if you'd like a more robust check)